README.md

```diff
@@ -8,6 +8,7 @@
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](https://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/mag.svg)](https://hub.docker.com/r/nfcore/mag)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4068515.svg)](https://doi.org/10.5281/zenodo.4068515)
```

Comment:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3589527.svg)](https://doi.org/10.5281/zenodo.3589527)
```

Reply:
Both actually resolve to the latest zenodo record: https://zenodo.org/record/4068515

Reply:
but yeah, the second one should give out all the versions, so that's the one we want